test/e2e/common/host_path.go

```diff
@@ -65,8 +65,12 @@ var _ = ginkgo.Describe("[sig-storage] HostPath", func() {
 	ginkgo.It("should support r/w [NodeConformance]", func() {
 		filePath := path.Join(volumePath, "test-file")
 		retryDuration := 180
+		path := "/tmp"
```

Comment:
Why is this needed? Was this working before on Windows?

Reply:
good point. it was working before. I realized this is related to my change. My code cannot handle path format like this /tmp/foo.., I think I should use normalizeWindowsPath before evaluating.

Reply:
I did a small test in Linux too, filepath.EvalSymlinks(base) returns the lowest layer symlink target